e2e/aws/rds_test.go

```diff
@@ -464,17 +464,18 @@ func connectAsRDSMySQLAdmin(t *testing.T, ctx context.Context, instanceID string
 	const dbName = "mysql"
 	info := getRDSAdminInfo(t, ctx, instanceID)
 
-	opt := func(conn *mysqlclient.Conn) {
+	opt := func(conn *mysqlclient.Conn) error {
 		conn.SetTLSConfig(&tls.Config{
 			ServerName: info.address,
 			RootCAs:    awsCertPool.Clone(),
 		})
+		return nil
 	}
 	endpoint := fmt.Sprintf("%s:%d", info.address, info.port)
 	conn, err := mysqlclient.Connect(endpoint, info.username, info.password, dbName, opt)
 	require.NoError(t, err)
 	t.Cleanup(func() {
-		_ = conn.Close()
+		_ = conn.Quit()
```

Comment:
Our old [patch](https://github.com/gravitational/go-mysql/pull/3/files) sends `COM_QUIT` in `conn.Close`. Now the driver provides a `Quit` function that does a similar thing so switching to `Quit` where I can.